.circleci/config.yml

```diff
@@ -37,11 +37,11 @@ binary_common: &binary_common
     build_version:
       description: "version number of release binary; by default, build a nightly"
       type: string
-      default: ""
+      default: "0.5.1"
```

Comment:
Why just a bump in patch version? Is there only bug fixes within this release?

Reply:
There will be no major new features in this release, so I think it might be better to make it a minor release.

But given that we haven't had a 1.0 release of torchvision, we have been using the patch numbering for minor releases. At some point we should decide when to cut the 1.0.0 release and do proper MAJOR.MINOR.PATCH versioning though